### PR TITLE
Actors controller

### PR DESCRIPTION
**Pull request by lavendermistyco**
May be some logic errors. Feel free to add any suggestions

### CODE REVIEW

File: routes/faithRoutes.js [hunk 1]
```diff
@@ -25,11 +35,26 @@ expressRouter.route('/actors')
...
-  .put(async (req, res) => {
+
+  .put(async (req, res) => { // UPDATE ACTOR NAME in the possibility that actor changed their name
     try {
-      console.log(`${defaultMsg} PUT`);
-      res.json({message: `${defaultMsg} PUT`});
+      const actors = await db.sequelizeDB.query(getTable('actors'), {
+        type: sequelize.QueryTypes.SELECT
+      });
+      const actorName = `SELECT * FROM actors WHERE actor_name = "${req.body.actor_name}"`;
```
Comment (dannyely) [inline routes/faithRoutes.js:45]: I think you need a second argument for this put

maybe have this method take two args of actor_name and new_actor_name

**Comment by dannyely** [comment]
One thing that would be helpful would to write out the arguments for the methods at the top of them, might make testing easier.  So PUT could have // takes args of actor_id etc

**Comment by dannyely** [comment]
Everything looks great!  don't see any issues

**Review by dannyely — APPROVED** [review]

**Review by uaryal — COMMENTED** [review]
Nice!

**Review by uaryal — COMMENTED** [review]
Looks great!

**Review by uaryal — APPROVED** [review]

**Pull request merged**